api-registration/server/index.js

```diff
@@ -0,0 +1,66 @@
+require('dotenv/config');
+const pg = require('pg');
+const argon2 = require('argon2'); // eslint-disable-line
+const express = require('express');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+
+const db = new pg.Pool({
+  // eslint-disable-line
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+
+app.post('/api/auth/sign-up', (req, res, next) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    throw new ClientError(400, 'username and password are required fields');
+  }
+
+  /* your code starts here */
+  const pass = argon2.hash(password);
+
+  pass.then(result => {
+    const hashedPassword = result;
+    const params = [username, hashedPassword];
```

Comment:
Here you could have named the parameter `hashedPassword` instead of creating a separate variable. For example:
```JAVASCRIPT
pass.then(hashedPassword => {
  const params = [username, hashedPassword];
```